src/kibana/directives/css_truncate.js

```diff
@@ -16,12 +16,14 @@ define(function (require) {
           'word-break': 'break-all',
         });
 
-        if (!_.isUndefined(attrs.cssTruncateExpandable)) {
-          $elem.css({'cursor': 'pointer'});
-          $elem.bind('click', function () {
-            $scope.toggle();
-          });
-        }
+        $timeout(function () {
+          if (!_.isUndefined(attrs.cssTruncateExpandable) && $elem[0].offsetWidth < $elem[0].scrollWidth) {
```

Comment:
This should probably be a watcher that can trigger multiple times and isn't subject to race conditions.